src/test/java/com/ddudu/user/service/FollowingServiceTest.java

```diff
@@ -217,7 +219,7 @@ class 팔로잉_수정_테스트 {
     }
 
     @Test
-    void 존재하지_않는_아이디면_실패한다() {
+    void 존재하지_않는_사용자면_실패한다() {
       // given
       long invalidId = faker.random()
           .nextLong();
```

Comment:
```suggestion
          .nextLong(Long.MAX_VALUE);
```

통일시켜줘도 좋을 것 같아요! 근데 급한 건 아니니까 인재님 마음대로 하셔도 될 것 같습니다!